website/docs/Examples/AgentChat.md

```diff
@@ -39,3 +39,8 @@ Links to notebook examples:
 5. **Agent Teaching and Learning**
    - Teach Agents New Skills & Reuse via Automated Chat - [View Notebook](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_teaching.ipynb)
    - Teach Agents New Facts, User Preferences and Skills Beyond Coding - [View Notebook](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_teachability.ipynb)
+
+6. **Multi-Agent Chat with OpenAI Assistants in the loop**
```

Comment:
are we missing the most basic one?

Reply:
https://github.com/microsoft/autogen/blob/main/notebook/agentchat_oai_assistant_twoagents_basic.ipynb

Reply:
Added. Thanks!